project/db/postgresql.conf

```diff
@@ -327,8 +327,8 @@ effective_cache_size = {{ salt['pillar.get']('postgresql_config:effective_cache_
                     # 0 disables.
 
 # These are relevant when logging to syslog:
-#syslog_facility = 'LOCAL0'
-#syslog_ident = 'postgres'
+syslog_facility = 'LOCAL0'
+syslog_ident = 'postgres'
```

Comment:
Never mind, we'll probably be routing all the syslog messages to something like papertrail and can search and filter them there.